ember-app/tests/unit/mixins/cards/card-move-test.js

```diff
@@ -0,0 +1,81 @@
+import Ember from 'ember';
+import CardMoveMixin from 'app/mixins/cards/card-move';
+
+import {
+  moduleFor,
+  test
+} from 'ember-qunit';
+
+var sut;
+
+function createIssueData(){
+  return Ember.Object.create({
+    _data: { order: 2.5 },
+    data: Ember.Object.create({ id: 123, title: 'test' })
+  });
+}
+
+function minMaxZeroOrderFixAlgorithm(issue){
```

Comment:
It strike me as odd to repeat this code here. Is there somewhere more accessible we could put it to reuse?